app/templates/direct-award/tell-us-about-contract.html

```diff
@@ -40,9 +40,69 @@ <h1 class="govuk-heading-xl">Tell us about your contract</h1>
       <form method="POST" action="{{ url_for('direct_award.tell_us_about_contract', framework_family=framework.family, project_id=project.id, outcome_id=outcome_id) }}" class="tell-us-about-contract-form" novalidate>
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
 
-        {{ form.start_date}}
+        {{ govukDateInput ({
+          "id": "input-start_date",
+          "namePrefix": "start_date",
+          "fieldset": {
+            "legend": {
+              "text": form.start_date.question,
+              "classes": "govuk-fieldset__legend--m"
+            }
+          },
+          "hint": {
+            "text": form.start_date.hint
+          } if form.start_date.hint,
+          "errorMessage": errors.start_date.errorMessage if errors.start_date.errorMessage,
+          "items": [
+            {
+              "classes": "govuk-input--width-2" + (' govuk-input--error' if errors.start_date.errorMessage),
```

Comment:
Hmm, it doesn't like this, try
```suggestion
              "classes": "govuk-input--width-2" ~ (' govuk-input--error' if errors.start_date.errorMessage),
```

Reply:
Ah yes, of course!